lab/lab2/js/part2-app-state.js

```diff
@@ -67,6 +88,7 @@ var removeMarkers = function() {};
  Leaflet setup - feel free to ignore this
 ===================== */
 
+
```

Comment:
![image](https://user-images.githubusercontent.com/8103418/53309482-0dc7b900-3876-11e9-8253-e3e693ba320e.png)


Code does not run..